metomi/rose/apps/rose_bunch.py

```diff
@@ -17,7 +17,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------------
-"""Builtin application: rose_bunch: run multiple commands in parallel."""
+# Builtin application: metomi.rose_bunch: run multiple commands in parallel.
```

Comment:
Why has this docstring become a comment?

Reply:
Proab'l an attempt to keep line len down. Fixed.